src/constant.h

```diff
@@ -7,594 +7,598 @@
 // Global constants used by Moria
 // clang-format off
 
-#define MAX_UCHAR         255
-#define MAX_SHORT       32767   // maximum short/long signed ints
-#define MAX_LONG   0x7FFFFFFFL
+#pragma once
+
+#include "headers.h"
```

Comment:
I'd still like to come to a decision on this. Unless they are causing compile/runtime issues, then we should remove them.

Can you let me know what problems you're having? Otherwise please go ahead and delete them. Thanks.